src/config/android/findManifest.js

```diff
@@ -0,0 +1,16 @@
+const glob = require('glob');
+const path = require('path');
+/**
+ * Find an android application path in the folder
+ *
+ * @param {String} folder Name of the folder where to seek
+ * @return {String}
+ */
+module.exports = function findManifest(folder) {
+  const manifestPath = glob.sync(path.join('**', 'AndroidManifest.xml'), {
+    cwd: folder,
+    ignore: ['node_modules/**', 'Examples/**', 'examples/**'],
```

Comment:
We use the same for iOS so might be useful extracting to separate method to ensure all globs ignore the same stuff.